source/Personality.cpp

```diff
@@ -101,26 +101,28 @@ Personality::Personality() noexcept
 
 
 
-void Personality::Load(const DataNode &node)
+void Personality::Load(const DataNode &node, int tokensToSkip)
 {
-	bool add = (node.Token(0) == "add");
-	bool remove = (node.Token(0) == "remove");
+	bool add = (node.Token(tokensToSkip) == "add");
+	bool remove = (node.Token(tokensToSkip) == "remove");
 	if(!(add || remove))
 		flags = 0;
-	for(int i = 1 + (add || remove); i < node.Size(); ++i)
+	for(int i = 1 + (add || remove) + tokensToSkip; i < node.Size(); ++i)
 		Parse(node, i, remove);
 
 	for(const DataNode &child : node)
 	{
 		if(child.Token(0) == "confusion")
 		{
 			if(add || remove)
-				child.PrintTrace("Error: Cannot \"" + node.Token(0) + "\" a confusion value:");
+				child.PrintTrace("Error: Cannot \"" + node.Token(tokensToSkip) + "\" a confusion value:");
 			else if(child.Size() < 2)
 				child.PrintTrace("Skipping \"confusion\" tag with no value specified:");
 			else
 				confusionMultiplier = child.Value(1);
 		}
+		else if(child.Size() == 3 && child.Token(0) == "grace" && child.Token(1) == "period")
```

Comment:
here you could have done this instead
```suggestion
		else if(child.Token(0) == "grace period")
```
no need to apply this, I'm just using this as an example to show you we can have multiple words in a single token, given they can be separated by `""`

Reply:
That is generally not how the Endless Sky txt works, and people objected to that in the "to display" functionality, so I've been keeping it as separate words.

Reply:
thats smth different entirely
because to display could be expanded to many different things; to highlight; to log or whatever

grace does not make sense with anything else than period

Reply:
There are no keyword tokens in Endless Sky with spaces in them, and I'm not going to add the first one.

Reply:
well get rid of grace period anyway

Reply:
also do take a look at government.cpp its full of 2 words token, there even is a 3 word one

Reply:
The player needs more than one frame to react to the government changing, and I don't want to restrict the engine to a hard-coded value.

Reply:
just remove the target when this happens

Reply:
What about the escorts? Them too?
It sounds obnoxious, but it may be the only safe way.

Reply:
no, not escorts, they already stop shooting when it'll be disabled, except if you give them the order again